inspirehep/modules/editor/views.py

```diff
@@ -216,6 +217,17 @@ def get_rt_queues():
     return jsonify(tickets.get_queues())
 
 
+@blueprint.route('/manual_merge', methods=['POST'])
+@editor_use_api_permission.require(http_exception=403)
+def start_manual_merge():
+    """Initiate manual merge on two records."""
+    assert request.json['head_recid']
+    assert request.json['update_recid']
```

Comment:
We need to do more validation than this: with the current code any user who can use the API can merge any two records, even if they can't read/edit them!

Reply:
What is the current best-practice to protect a view to catalogers? This is definitively a very common use case?

I see the editor uses: `@editor_use_api_permission.require(http_exception=403)`. As ugly as it is, can we re-use it?

Reply:
There is also `@editor_permission` which sounds a good candidate.